### PR TITLE
[tests-only][full-ci] remove webUI test related to user redirection to login page after the logged in user is deleted

### PR DESCRIPTION
**Pull request by PrajwolAmatya**
## Description
In this PR,
- removed the webUI test scenario related to user redirection to login page when logged in user is deleted

## Related Issue

- https://github.com/owncloud/web/issues/10259

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests
- [ ] Documentation
- [ ] Maintenance (e.g. dependency updates or tooling)

### CODE REVIEW

**Review by SagarGi — COMMENTED** [review]
LGTM :+1:

**Review by grgprarup — COMMENTED** [review]
LGTM :+1:

**Review by ScharfViktor — APPROVED** [review]
LGTM

File: tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md [hunk 1]
```diff
@@ -7,8 +7,3 @@ Please follow this format for the actual expected failures.
 Level-3 headings should be used for the references to the relevant issues. Include the issue title with a link to the issue in GitHub.
 
 Other free text and markdown formatting can be used elsewhere in the document if needed. But if you want to explain something about the issue, then please post that in the issue itself.
-
-### [Exit page re-appears in loop when logged in user is deleted](https://github.com/owncloud/web/issues/4677)
-
-- [webUILogin/openidLogin.feature:50](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUILogin/openidLogin.feature#L50)
-- [webUILogin/openidLogin.feature:60](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUILogin/openidLogin.feature#L60)
```
Comment (saw-jan) [inline tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md:14]: do we have the test coverage for this bug report in unit/e2e/api? just to be sure that we are not decreasing the coverage

CC @ScharfViktor

Comment (saw-jan) [reply tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md:14]: It is a web issue so I think adding coverage in API won't do it.

Comment (ScharfViktor) [reply tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md:14]: yes, but it's low priority bug. I wouldn't write tests for the p4 prio bugs

**Review by saw-jan — APPROVED** [review]

**Pull request merged**